blocks/context/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External depencencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+
+/**
+ * This is a stopgap until React 16.3 is released with the new context API.
```

Comment:
Can we just pull in the React alpha? I'd personally be fine with it. (We ran React 16.0.0 alpha for many months)

Reply:
That would be much better. I'll do that.

Reply:
Well, I'll do that if our approach to autocompletion leaves us with contextually defined autocompleters.

Reply:
I updated https://github.com/WordPress/gutenberg/pull/5029 which contains all required changes to bring in React 16.3. You can cherry pick the initial commit from this PR or merge the full PR as it is. I tried to make it ready to merge with all the required docs updates and so on. Just letting you know. There is also https://www.npmjs.com/package/create-react-context polyfill that we could use temporarily.